compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/input/pointer/SyntheticEventSender.skiko.kt

```diff
@@ -191,7 +192,8 @@ internal class SyntheticEventSender(
     }
 
     private fun PointerInputEvent.pressedIds(): Sequence<PointerId> =
-        pointers.asSequence().filter { it.down }.map { it.id }
+        pointers.fastMapNotNull { if (it.down) it.id else null }.asSequence()
```

Comment:
```suggestion
private fun PointerInputEvent.pressedIds(): List<PointerId> =
        pointers.fastMapNotNull { if (it.down) it.id else null }
```
No one downstream uses this as a sequence. They immediately call `toSet()` most of the time or perform a difference operation.

Reply:
Done